v1.0/benchmarks/redis_callback_class.py

```diff
@@ -0,0 +1,56 @@
+from factor_graph import RedisCallbackClass
+from state import RedisNodeStateStore
+from node_update_functions import ALGORITHM_TO_UPDATE_FUNCTIONS
+from redis import Redis
+import timeit
+
+def timer(function_to_time):
+    def function_timer(*args, **kwargs):
+        setup()
+        t = timeit.Timer(function_to_time)
+        print(function_to_time.__name__, " benchmark: ", t.timeit(10))
```

Comment:
since we're not timing the print fuction, would it be better to put print outside of takedown? just wanna make sure this is in the right place

Reply:
I don't think `print` is included in the timing itself - but happy to move it out to be more explicit.